recipe/meta.yaml

```diff
@@ -24,13 +25,16 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6.2
-    - astroid >=2.9.0,<2.10
+    - python
+    - astroid >=2.11.6,<2.12.0
     - colorama  # [win]
```

Comment:
`colorama` should be pinned as `>=0.4.5`, see https://github.com/PyCQA/pylint/blob/566ffca72a1a08c9e7ec14383c4710d64165c592/setup.cfg#L55